static/js/components/inputs/inputs_test.js

```diff
@@ -41,36 +46,36 @@ describe('Profile inputs', () => {
       {value: 'f', label: 'Female'},
       {value: 'o', label: 'Other/Prefer not to say'}
     ];
-    let extendedOptions = [
+    let letterOptions = [
       {value: 'a', label: 'Option 1'},
       {value: 'b', label: 'Option 2'},
       {value: 'c', label: 'Option 3'},
       {value: 'd', label: 'Option 4'},
-      {value: 'e', label: 'Option 5'}
+      {value: 'e', label: 'Option 5'},
+      {value: 'f', label: 'Option 6'},
+      {value: 'g', label: 'Option 7'},
+      {value: 'h', label: 'Option 8'},
+      {value: 'i', label: 'Option 9'},
+      {value: 'j', label: 'Option 10'},
+      {value: 'k', label: 'Option 11'}
     ];
 
     let renderGenderSelectField = () => {
-      let selectProps = Object.assign({},
-        inputProps,
-        {
-          keySet: ['gender'],
-          label: "Gender",
-          options: genderOptions
-        }
-      );
-      return renderTestComponent(SelectField, selectProps);
+      _.extend(inputProps, {
+        keySet: ['gender'],
+        label: "Gender",
+        options: genderOptions
+      });
+      return renderTestSelectField();
     };
     
-    let renderExtendedSelectField = () => {
-      let selectProps = Object.assign({},
-        inputProps,
-        {
-          keySet: ['extended'],
-          label: "Extended",
-          options: extendedOptions
-        }
-      );
-      return renderTestComponent(SelectField, selectProps);
+    let renderLetterSelectField = () => {
+      _.extend(inputProps, {
```

Comment:
Why not `Object.assign(inputProps, {...})` then? At least for our purposes it's the same function, right?